<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qHTTPing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="options">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>511</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>URL:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="URL">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>451</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ping interval:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="interval">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>66</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="skipPeerVerify">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>skip peer verification</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="skipHostNameVerify">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>skip hostname verification</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="doHeadRequest">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>HEAD</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="pipelining">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>pipelining</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="httpVersion">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>HTTP2</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="timeout">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>66</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Timeout:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="GO">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EXIT">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(C) 2018 by folkert@vanheusden.com</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="results">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>511</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="averages">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Result:</string>
     </property>
    </widget>
    <widget class="QLabel" name="counts">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>421</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
